packages/web/components/Technology/Details/Header.js

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
 import styled, { css } from 'styled-components';
+import { useTranslation } from 'react-i18next';
 import { Button } from '../../Button';
-import { useTechnology } from '../../../hooks';
+import { useTechnology, useAuth, useModal } from '../../../hooks';
 import { Likes, Share } from '../../Card';
 import ImagesCarousel from './ImagesCarousel';
 
 const Header = () => {
 	const { technology, implementationCosts } = useTechnology();
+	const { openModal } = useModal();
+	const { user } = useAuth();
+	const { t } = useTranslation(['common']);
+
+	const handleClick = ({ target: { name } }) => {
```

Comment:
Awesome trick for easier future feature expansion 🚀

Reply:
Good man